<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondDialog</class>
 <widget class="QDialog" name="SecondDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI - Usuario</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="tx_label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>30</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_llamar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: None;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_contactos">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>138</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #6a6ea9;
border-radius: 10px;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="itemAlignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fondo_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>diseno/Fondo2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="usuario">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>26</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>diseno/Usuario.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>diseno/B7.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tx_label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>118</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Nueva Llamada</string>
   </property>
  </widget>
  <zorder>fondo_2</zorder>
  <zorder>tx_label</zorder>
  <zorder>Btn_llamar</zorder>
  <zorder>list_contactos</zorder>
  <zorder>usuario</zorder>
  <zorder>usuario_2</zorder>
  <zorder>tx_label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
